torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -639,6 +645,12 @@ def perfect_match_inputs(
                 diagnostic.info("The function is not a nearest match candidate.")
                 is_perfect_match = False
 
+            # Try to remove attributes with default values.
+            while len(function_attributes) > len(self.attributes):
+                last_attr = list(function_attributes.keys())[-1]
```

Comment:
Is it safe to always remove the last attribute?

Reply:
From matcher perspective, ignoring the default attributes in matching would mess up the differentiation between the overloads that depends on the kwargs difference. It feels buggy to add it right into current matcher. I stil suggest xfail/skip and update split for this case, and have this improvement (PR) by ourselves.